### ISSUE TITLE
[Angular - NgRx - SCSS] Create gists panel

### ISSUE THREAD

**Issue by lindakatcodes** [thisdot/starter.dev-github-showcases#186]
# Create gists panel

## Background

As a user, I should see an unordered list of my gists on the main home page.

## Acceptance

- [x] Gist title
- [x] unordered list of user gists
- [x] clicking a gist opens that file in a new tab

## Notes

**Comment by lindakatcodes** [comment]
I'd double check on this, but I think this is already implemented in Daian's PR.

**Comment by Reshurum** [comment]
> I'd double check on this, but I think this is already implemented in Daian's PR #319

Correct

**Comment by lindakatcodes** [comment]
We should still leave this ticket open - there's a few small things to fix once Daian's PR is merged that will be easier to fix in a new PR instead of continuing on the single large one.

Still to do:
- [ ] accessibility: Consider using semantic HTML elements <ul> and <li> here. 
- [ ] styling: needs mobile styling (gist section should move to bottom of page on small screens, see next-react-query deployed app for example)

For reference if needed: https://github.com/thisdot/starter.dev-github-showcases/pull/319#discussion_r912118267
